pkg/cli/logging.go

```diff
@@ -6,13 +6,36 @@ import (
 	"os"
 )
 
-func InitGlobalLogger() *zap.Logger {
+const (
+	DebugLevel = "debug"
+	InfoLevel  = "info"
+	WarnLevel  = "warn"
+	ErrorLevel = "error"
+	FatalLevel = "fatal"
+	PanicLevel = "panic"
+)
+
+var stringToLevel = map[string]zapcore.Level{
+	DebugLevel: zapcore.DebugLevel,
+	InfoLevel:  zapcore.InfoLevel,
+	WarnLevel:  zapcore.WarnLevel,
+	ErrorLevel: zapcore.ErrorLevel,
+	FatalLevel: zapcore.FatalLevel,
+	PanicLevel: zapcore.PanicLevel,
+}
+
+func InitGlobalLogger(level string) *zap.Logger {
+	zapLevel, ok := stringToLevel[level]
```

Comment:
can we make level string lowercase here? Just in case someone decides to pass the log level in capital letters.